### PR TITLE
feat(tools) add script to audit all packages

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Lerna does not (yet) have a command to run `npm audit` or `npm audit fix` on all packages in a repo.  This script does that, ensuring that all the packages in `lerna.json` get audited.  I decided on `npm audit fix`, but if we want `npm audit` as well, I can add that in.

This should make #36535 a bit easier to solve.

### CODE REVIEW

**Comment by raisedadead** [comment]
I think you are able to do `lerna exec npm audit` and  `lerna exec npm audit fix`

**Comment by ojeytonwilliams** [comment]
Huh.  I assumed that, since Lerna has an open issue about auditing, it couldn't be that easy.  Shows what I know.

Do you think it's still worth creating a couple of npm scripts for that, in case I'm not the only one that makes this mistake?  Even though my script isn't needed.

**Comment by raisedadead** [comment]
Hmm, I think we should be keeping the `fix` part from the script as a separate command or something.

More often than not, I have seen even in range updates break the setup. 🤷‍♂

**Comment by ojeytonwilliams** [comment]
How about these?  It's a bit slow with concurrency 1, but I guess we aren't going to be running it all the time.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "private": true,
   "version": "0.0.1",
   "scripts": {
+    "audit": "lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit'",
```
Comment (raisedadead) [inline package.json]:
```suggestion
    "audit": "npm audit && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit'",
```

File: package.json [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
+    "audit:fix": "lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit fix'",
```
Comment (raisedadead) [inline package.json]:
```suggestion
    "audit:fix": "npm audit fix && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit fix'",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Lerna will not cover the root of the repo.

**Comment by ojeytonwilliams** [comment]
Good point.  Thanks!

**Review by raisedadead — APPROVED** [review]

**Pull request merged**